raiden-network/light-client
# Raiden account as default connect method

**Thank you for submitting this pull request :)**

Resolves #1302  

**Short description**


**Definition of Done**


**Steps to manually test the change (dApp)**

1. Serve dApp
2. Click connect
3. Confirm usage of raiden account in connect dialog
4. Disconnect
5. Toggle between main and raiden account in the settings menu (identicon => settings => use raiden account)
